src/components/Card/Card.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from "react";
+import "./Card.css";
+import noCardImg from "../../assets/basic-card.png";
+import kebabIcon from "../../assets/kebab-icon.svg";
+import elapsedTime from "../../utils/elapsedTime";
+import CardPopover from "../CardPopover/CardPopover";
+
+function Card({ card, changeLinkDeleteSelect, changeFolderAddSelect }) {
+  const [showPopover, setShowPopover] = useState(false);
+
+  const handleKebabClick = (e) => {
+    e.preventDefault();
+    setShowPopover(!showPopover);
```

Comment:
```suggestion
    setShowPopover((prev) => !prev);
```
이렇게하는게 더 안전할 것 같은데 왜일까요~?